src/components/cluster/new/aws_instance_type_selector.js

```diff
@@ -103,7 +103,7 @@ class AWSInstanceTypeSelector extends React.Component {
 
   render() {
     return (
-      <div className='col-4'>
+      <div style={{ maxWidth: '200px' }}>
```

Comment:
There might be a good reason in this case, but generally, would like to avoid inline styles.